src/Practices/KW/SparseVector.cs

```diff
@@ -0,0 +1,89 @@
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Security;
+
+namespace MySparseVector;
+public class SparseVector
+{
+    Dictionary <int, double> elements;
+
+    public SparseVector()
+    {
+        elements = new Dictionary<int, double>  ();
+    }
+    
+    public void SetValue(int index, double value)
+    {
+        if (value != 0)
+        {
+            elements[index] = value;
+        }
+        else if (elements.ContainsKey(index))
+        {
+            elements.Remove(index);
+        }
+    }
+
+    public double GetValue(int index)
+    {
+        if (elements.ContainsKey(index))
+        {
+            return elements[index];
+        }
+        else
+        {
+            return 0;
+        }
```

Comment:
```suggestion
        return elements.ContainsKey(index) ? elements[index] : 0;
```
Это то же самое, но в восемь раз короче